db/schema.rb

```diff
@@ -53,11 +53,14 @@
     t.index ["slug"], name: "index_communities_on_slug", unique: true, using: :btree
   end
 
-  create_table "index_choices", force: :cascade do |t|
-    t.integer  "index_id"
-    t.integer  "choice_id"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+  create_table "indices", force: :cascade do |t|
+    t.datetime "finalized_at"
+    t.integer  "category_id"
```

Comment:
Question.  Do we want to use category_id to make setting up the relations easier, or should we use awrvi_version, and set the class_name, etc.? Or is there a third, better, way?